_harp/about/_company.md

```diff
@@ -1,11 +1,19 @@
 # About us
 
-Founders & Coders C.I.C. runs the only free full-time coding course in the UK. We have developed a highly effective peer-led model of learning which, after only two years of operation, attracts exceptional women and men from all over the world. 30/31 of our most recent graduates are working as software developers.
+Founders & Coders runs a well-established coding camp in London. We are delighted to announce that in 2017 we will be opening our first international campus in the Arab Israeli city of Nazareth.
 
-After completing the course, students are encouraged to join our freelance community and to contribute to a growing array of community activities, including the course itself.
+We follow a highly-effective peer-led approach to learning and we do not employ any teaching or administrative staff. Our low cost of operation allows us to offer our training without charging any fees to our students.
 
-We support ourselves entirely through voluntary contributions from our alumni, through placement fees from employers, and through our close working relationship with [dwyl](http://www.dwyl.io/), which provides freelance experience to many of our graduates.
+The programme is project-based. Starting with a 10-week foundation course, our students then go on to work with charities and early-stage startups to find creative technical solutions to real-world problems.
+
+After students have completed four months of training, they continue to work with us, usually for at least two months, on a variety of projects provided through our partners, while mentoring the next cohort of students.
+
+We cover a curriculum of test-driven development using JavaScript and Node.js, we follow agile software development practices and we train the charities and startups that we work with in the lean methods that we use.
+
+Nearly every one of our graduates finds a software development role. The only drop-out we have had out of 64 students in the 12 months to September 2016 was a student who received a job offer before he completed the programme.
+
+Our coding camps are an expression of our belief in the power of democratic pedagogy to build bridges between people and to help provide creative solutions to the world’s economic and social challenges.
+
+These programmes would not be possible without the support of [dwyl](http://www.dwyl.io/), who provide freelance employment to many of our graduates, [CAST](http://www.wearecast.org.uk/), who we partner with on charity projects, and [UK Israel Tech Hub](http://www.ukisraelhub.com/) who are supporting our work in Nazareth.
 
```

Comment:
Maybe also add a mention to the directors and mentors and link to the relevant page on the site?